db/migrate/20190225171920_create_evenements.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvenements < ActiveRecord::Migration[5.2]
+  def change
+    create_table :evenements do |t|
+      t.string :type_evenement
+      t.text :description
```

Comment:
On pourrait le changer en `data` ou équivalent en francais et le passer en `jsonb` histoire de pouvoir faire des requêtes sympa https://www.postgresql.org/docs/9.4/datatype-json.html

Reply:
l'équivalent français serait `données`, ok pour le jsonb, il y a des infos ici : https://edgeguides.rubyonrails.org/active_record_postgresql.html#json-and-jsonb